iib/workers/config.py

```diff
@@ -60,6 +60,7 @@ class Config(object):
     iib_total_attempts: int = 5
     iib_retry_delay: int = 5
     iib_retry_jitter: int = 5
+    iib_retry_multiplier: int = 5
```

Comment:
We probably want to add this to README.md

Reply:
Thanks, please check if the description of the constant in the README.md is okay.